website/www/site/content/en/contribute/release-guide.md

```diff
@@ -1134,12 +1134,29 @@ Create and push a new signed tag for the released version by copying the tag for
     git tag -s "$VERSION_TAG" "$RC_TAG"
     git push upstream "$VERSION_TAG"
 
-After the tag is uploaded, publish the release in the Github UI.
-1. Navigate to `https://github.com/apache/beam/releases/tag/$VERSION_TAG`.
-1. Click the "Edit tag" button.
-1. Give the release a title, such as `Beam 2.21.0`.
-1. For the release description, copy the current version's changes from `CHANGES.md`. (You may want to touch up the formatting a bit.)
-1. Click the "Publish release" button.
+After the tag is uploaded, publish the release by calling the Github API (Make sure that the `GITHUB_TOKEN` variable is set):
```

Comment:
There's an earlier step in the release guide that has us [create a github token and write it to script.config](https://github.com/apache/beam/blob/59b87d87c3cedf89fda11f5cf31c066c852064d6/website/www/site/content/en/contribute/release-guide.md#run-automation-script-verify_release_buildsh). It would be nice if this were a script in release/src/main/scripts/ and could read the same token from script.config.